### PR TITLE
Manual Sign In with Email and Password.

### PR DESCRIPTION
**Pull request by pandilladivya**
Integrated Firebase to the application.

I've added 3 sample users to the database.
The following is the screenshot.

screen shot 2017-06-18 at 11 58 42 pm

**Demo**

ezgif com-video-to-gif 4

### CODE REVIEW

**Comment by pandilladivya** [comment]
@sandarumk @HimanshiJain 
Kindly Review.
Thanks

**Comment by HimanshiJain** [comment]
It works well :) But how do you get to the next acitivity? Shouldn't it direct you to the home activity if you have successfully logged in through any of the options?

File: app/src/main/java/vola/systers/com/volunteers_android/activities/SignInActivity.java [hunk 1]
```diff
@@ -40,12 +48,19 @@
...
     private GoogleApiClient mGoogleApiClient;
     private ProgressDialog mProgressDialog;
     CallbackManager callbackManager;
+    private static final String TAG = SignInActivity.class.getSimpleName();
+
+    private FirebaseAuth mAuth;
+    private FirebaseAuth.AuthStateListener mAuthListener;
+
+    private EditText emailText,passwordText;
+    private Button login;
```
Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/volunteers_android/activities/SignInActivity.java]: Okay. Will rename :)

**Comment by sandarumk** [comment]
I will merge this now. Please make the httphandler a util class with static methods as I mentioned in the earlier PR.

**Comment by pandilladivya** [comment]
@sandarumk Sure :)

**Pull request merged**